index.bs

```diff
@@ -729,6 +729,8 @@ When this method is invoked, the user agent MUST execute the following algorithm
     1. [=map/Set=] |authenticatorExtensions|[|extensionId|] to the [=base64url encoding=] of |authenticatorExtensionInput|.
 
 1. Let |collectedClientData| be a new {{CollectedClientData}} instance whose fields are:
+    : {{CollectedClientData/type}}
+    :: The string `webauthn.create`.
```

Comment:
per https://heycam.github.io/webidl/#prod-string string literals are in double-quotes